content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -137,20 +153,55 @@ use, see `man 7 user_namespaces`.
 
 ## Set up a node to support user namespaces
 
-It is recommended that the host's files and host's processes use UIDs/GIDs in
-the range of 0-65535.
-
-The kubelet will assign UIDs/GIDs higher than that to pods. Therefore, to
-guarantee as much isolation as possible, the UIDs/GIDs used by the host's files
-and host's processes should be in the range 0-65535.
+By default, the kubelet assumes host's files and host's processes use UIDs/GIDs
+in the range of 0-65535. This is the case in most Linux distributions with the
+default installation. The kubelet will assign UIDs/GIDs higher than that to
+pods, so there is no overlap between UID/GIDs used by host's files and processes
+and the ones used by pods.
 
 Note that this recommendation is important to mitigate the impact of CVEs like
```

Comment:
```suggestion
The assumption and default setting for user and group ID ranges is important to mitigate
the impact of security vulnerabilities such as
```

Reply:
I guess what you want to fix here is that we don't say "recommended" anymore now? I think it is clear as it is, but changing it to this instead: 

```
Avoiding the overlap is important to mitigate....
```